pagopa/io-app
# feat: [IOCOM-455] Add bottom sheet for multiple F24

> [!warning]
> This PR depends on #5156 

## Short description
This PR adds the bottom sheet for multiple F24.

### Preview

| bottom sheet |
| --- |
| <video src="https://github.com/pagopa/io-app/assets/29163287/6e1c5390-b67f-443c-b4df-2a4d2b56abf2" width="250" /> |

## List of changes proposed in this pull request
- added `useF24BottomSheet` to display multiple F24
- updated langs

## How to test
Generate a SEND message with multiple F24 using `io-dev-api-server`. Check that the bottom sheet for multiple F24 is displayed correctly.
